R/label-placer.R

```diff
@@ -201,6 +203,44 @@ label_placer_manual <- function(breaks, x, y, theta) {
 }
 
 
+#' @rdname label_placer
+#' @export
+label_placer_middle <- function(rot_adjuster = angle_halfcircle_bottom(), ...) {
```

Comment:
CI still doesn't pass. It looks to me like the `...` argument here shouldn't be there. (The tests complain about an undocumented argument, but I think the bigger issue is that it doesn't belong here.)

You can run R check locally before making the next commit to make sure all issues are addressed.

Reply:
yes, this is the problem.